src/test/java/io/supertokens/test/session/api/MultitenantAPITest.java

```diff
@@ -424,4 +426,77 @@ public void testAccessTokensContainsTid() throws Exception {
             assertEquals(t2.getTenantId(), accessTokenInfo.payload.get("tId").getAsString());
         }
```

Comment:
add tests where multi tenancy is not enabled and we are calling these functions.